active_directory/setup.py

```diff
@@ -0,0 +1,127 @@
+# Always prefer setuptools over distutils
+from setuptools import setup
+# To use a consistent encoding
+from codecs import open
+from os import path
+
+import json
+import re
+
+here = path.abspath(path.dirname(__file__))
+
+def parse_req_line(line):
```

Comment:
This looks like the old-style wheel. If possible it would be nice to merge the new stuff with the newer polished wheels approach we can see in `disk` or `redis` where we use tox and pytest, and a bit of a cleaner `setup.py` as we read the requirements from `requirements.in` (used by tox) instead of parsing `requirements.txt`. The downside is two sources of truth for the requirements.